src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -24,757 +24,757 @@ public static partial class CpuMathUtils
         public static int GetVectorAlignment()
             => Avx.IsSupported ? Vector256Alignment : (Sse.IsSupported ? Vector128Alignment : FloatAlignment);
 
-        public static void MatTimesSrc(bool tran, bool add, AlignedArray mat, AlignedArray src, AlignedArray dst, int crun)
+        public static void MatrixTimesSource(bool transpose, bool add, AlignedArray matrix, AlignedArray source, AlignedArray destination, int stride)
```

Comment:
This is a partial class, and the `.netstandard.cs` side of the partial class needs to be updated as well.

BTW - are you compiling for .NET Core 3.0? I assume there are callers of this function that need to be updated.  I have `3.0` building instructions in this soon to be merged PR: https://github.com/dotnet/machinelearning/pull/1032

Reply:
Thanks, @eerhardt! I totally didn't realize it was a partial class. The `netstandard` class has been updated as well as the callers of the functions that were changed.

I'll mess with the .NET Core 3.0 part to make sure all that looks good.

Reply:
@eerhardt Tried to compile for .NET Core 3.0 and, with the VS route, it seemed that everything in the `CpuMath` library fails to compile.

I also tried through the command line and would get this error:
![image](https://user-images.githubusercontent.com/1578160/46373170-4b277200-c65b-11e8-9f72-a6876ef56747.png)

I'm sure I probably did a step wrong so I'll continue to mess with it, but I figured I'd give an update.

Reply:
Something is eating your `/` in `/p:Configuration=Release-Intrinsics` on the command line. What shell are you using? CMD? Powershell?

try specifying it with a `-` instead: `-p:Configuration=Release-Intrinsics`

Reply:
That worked better! I use bash on [Cmder](http://cmder.net/) for my terminal.

It looks like it builds for the most part, but I do get the error below. Even after running `dotnet restore` it still comes up. I wouldn't think it has anything to do with these changes, though.

```bash
C:\Users\jwood\Documents\Code\GitHub\machinelearning\Tools\dotnetcli\sdk\3.0.100-alpha1-009632\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(202,5): 
error NETSDK1004: Assets file 'C:\Users\jwood\Documents\Code\GitHub\machinelearning\bin\obj\AnyCPU.Release-Intrinsics\Microsoft.ML.CpuMath\project.assets.json' not found. 
Run a NuGet package restore to generate this file. [C:\Users\jwood\Documents\Code\GitHub\machinelearning\src\Microsoft.ML.CpuMath\Microsoft.ML.CpuMath.csproj]
```

Reply:
Do you have my latest code:  https://github.com/dotnet/machinelearning/pull/1032?  That should solve that problem.

Reply:
Great catch, @eerhardt! Turns out I did miss some updates from a previous change. Thanks!